format/gcm-stream-spec.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "AES GCM Stream Spec"
+url: gcm-stream-spec
+toc: true
+disableSidebar: true
+---
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+# AES GCM Stream (AGS) file format extension
```

Comment:
Is AGS a common alias for AES GCM? I don't think that we should introduce a new acronym.

Reply:
While AES GCM is a NIST standard, AES GCM Stream is a new concept introduced in Iceberg. This is a file encryption stream, based on the GCM properties, so the "AES GCM Stream" (and AGS for short references where needed) seem to be a reasonable naming; but there might be other options.